cumulusci/tasks/debug_logs.py

```diff
@@ -0,0 +1,135 @@
+import datetime
+import os
+import csv
+
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class TurnOnDebugLogs(BaseSalesforceApiTask):
+    task_docs = """
+    Turn on debugging information in Salesforce
+    """
```

Comment:
Please configure the new tasks in cumulusci.yml and run `cci task doc > docs/tasks.rst` to update the generated task reference docs.